controllers/serviceexport/unit_test.go

```diff
@@ -0,0 +1,31 @@
+package serviceexport_test
```

Comment:
as we discussed, could you please use the `serviceexport` package? and rename the filename to controller_test.go

Reply:
Done. Sorry that I missed that part.